src/sentry/static/sentry/app/components/dropdownAutoCompleteMultiRow.jsx

```diff
@@ -0,0 +1,86 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import CheckboxFancy from 'app/components/checkboxFancy';
+import space from 'app/styles/space';
+
+class DropdownAutoCompleteMultiRow extends React.Component {
+  static propTypes = {
+    checked: PropTypes.bool,
```

Comment:
Can you mark required props as required?

Reply:
yes I'm sorry. I sometimes forget that part! 😬😬😬